### PR TITLE
validate token

### PR DESCRIPTION
**Pull request by congtran8991**
Validate token api  when user login

### CODE REVIEW

File: projectname/src/api/validate/routes/validate.js [hunk 1]
```diff
@@ -9,5 +9,14 @@ module.exports = {
        middlewares: [],
      },
     },
+    {
+      method: 'POST',
+      path: '/token/decrypt',
```
Comment (nguyenhoaidanh) [inline projectname/src/api/validate/routes/validate.js:14]: /token/verify

File: projectname/src/extensions/users-permissions/content-types/user/schema.json [hunk 2]
```diff
@@ -79,8 +79,7 @@
     "phone": {
       "type": "biginteger",
       "unique": true,
-      "required": true,
-      "max": "11"
```
Comment (nguyenhoaidanh) [inline projectname/src/extensions/users-permissions/content-types/user/schema.json:83]: why remove

File: projectname/src/api/validate/routes/validate.js (hunk 1 above)
Comment (congtran8991) [reply projectname/src/api/validate/routes/validate.js]: done

File: projectname/src/extensions/users-permissions/content-types/user/schema.json (hunk 2 above)
Comment (congtran8991) [reply projectname/src/extensions/users-permissions/content-types/user/schema.json]: done

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**